lib/PuppeteerSharp/Puppeteer.cs

```diff
@@ -12,5 +12,10 @@ public class Puppeteer
         {
             return await new Launcher().LaunchAsync(options, chromiumRevision);
         }
+
+        public static async Task<Browser> ConnectAsync(ConnectOptions options)
```

Comment:
Could you add an Xml doc here?
Just copy these cool guys https://github.com/GoogleChrome/puppeteer/blob/v1.0.0/docs/api.md